cheatsheets_draft/Authorization_Cheat_Sheet.md

```diff
@@ -0,0 +1,47 @@
+# Authorization Cheat Sheet
+
+## Introduction
+
+Focus on definition and distinguishing it from authentication. Perhaps explore types of privilege escalation (horizontal and vertical). Not sure if we should briefly explore the different access control models (RBAC, MAC, DAC, etc) or just leave it out?
```

Comment:
I think that introducing MAC is just adding complication - it's not relevant to 99% of developers.

Reply:
I am definitely against describing them. Maybe mention Access Controls in the references section, since in no way is this going to make it "intriguing" and actionable if we mention non-relevant stuff to the CS.